subprojects/docs/src/docs/release/notes.md

```diff
@@ -39,6 +39,37 @@ For Java, Groovy, Kotlin and Android compatibility, see the [full compatibility
 
 TODO: Expand this section
 
+## Toolchains for JVM projects
+
+By default, Gradle uses the same Java version for running Gradle itself and building JVM projects. 
+
+This is not always desirable. Building projects with different Java versions on different developer machines and CI servers may lead to unexpected issues Additionally, you may want to build a project using a Java version that running Gradle is not compatible with.
+
+Before this release, it required [several steps](userguide/building_java_projects.html#compiling_and_testing_java_67) to configure a different Java version for compilation, testing, generating Javadoc, and executing applications.  
+
+This release introduces a new concept called Java Toolchains to Gradle. A Java Toolchain (from now on referred to simply as toolchain) is a set of tools (e.g. a java compiler), taken from a local JRE/JDK installation and used to configure the build.
+
+But instead of manually specifying this for the various tasks, toolchains provide a holistic set of requirements that define what the build needs.
+
+Setting up a toolchain can be done through the `java` extensions:
+
+```
+java {
+    toolchain {
+        languageVersion = JavaLanguageVersion.of(14)
+    }
+}
+```
+
+Compile tasks use `javac` as their compiler, test and exec tasks may use the `java` command while `javadoc` will be used to generate documentation. All taken from the selected toolchain above.
```

Comment:
Let's make the first sentence consistent so it does not mix "use", "may use" and "will be used" for equivalent items.